### DISCUSSION TITLE
Nextcloud Updates <> App eID-Login

### DISCUSSION THREAD

**Discussion by majo1989** [nextcloud/all-in-one#2992] [Questions] [answered]
Hello,
with every Nextcloud update, the individual apps are first uninstalled/deactivated and then reinstalled/activated after the update.

During this process, the eID-Login app is also subject to this process. The problem here is that the service provider etc. has to be stored again afterwards. After that, every user has to link their ID card to the Nextcloud again if they use it to log in. 

Is it possible to generate an exception for this app so that this setup process does not have to be repeated every time?

Thanks.

**Comment by szaimen** [comment, marked as answer]
Hi, as with any other Nextcloud app it is expected that disabling the app does not remove any data. So this should rather be fixed in the eid-login app.
